docs/cli/migrate.md

```diff
@@ -0,0 +1,39 @@
+---
+id: migrate
+title: "Migrating from Botkube Open Source"
+sidebar_position: 2
+---
+
+## Migrating from Botkube Open Source
```

Comment:
WDYT about "migrating to Cloud" instead from "OS"? At the end Botkube connected to Cloud is still the open source one. Something like:

```suggestion
id: migrating-installation-to-botkube-cloud
title: "Migrating installation to Botkube Cloud"
sidebar_position: 2
---

## Migrating installation to Botkube Cloud
```